packages/preferences-api/src/common/preference-service.ts

```diff
@@ -49,7 +50,7 @@ export class PreferenceService implements Disposable {
 
         this.resolveReady();
         for (const change of event.changes) {
-            this.onPreferenceChangedEmitter.fire(change);
+            this.onPreferenceChangedEmitter.fire(clone(change));
```

Comment:
It means there will be only one copy for all listeners? I am not sure how to handle it, I see 2 ways:
- instead of cloning, we do `Object.freeze` that clients will get a runtime error if they try to modify and will be forced to clone themselves
- or we don't send a value as a part of an event, clients should always use `get`